seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -1256,8 +1256,8 @@
       "id": "bad87fee1348bd9aedf08828",
       "title": "Create an Ordered List",
       "description": [
-        "HTML has a special element for creating <code>ordered lists</code>, or numbered-style lists.",
-        "Ordered lists start with a <code>&#60;ol&#62;</code> element. Then they contain some number of <code>&#60;li&#62;</code> elements.",
+        "HTML has another special element for creating <code>ordered lists</code>, or numbered lists.",
+        "Ordered lists start with an opening <code>&#60;ol&#62;</code> element, followed by any number of <code>&#60;li&#62;</code> elements. Finally, end the ordered list with a closing <code>&#60;/ol&#62;</code>",
```

Comment:
"Finally, end the ordered list with..." is imperative. I think what you mean to say is "Finally, ordered lists close with a </ol>"